content/creator/wearables-and-emotes/manage-collections/uploading-smart-wearables.md

```diff
@@ -0,0 +1,165 @@
+
+Save the price and you will be back to the list of wearables in your collection. When you click on the item, you will get its general info. Click on the button **_Preview in Editor_** to see it on the editor.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/13_click_item.png" width="900"/>
+
+# The Editor
+
+Once you click on **_Preview in Editor_**, you will have the editor open. You can edit all the info of your smart wearable, as well as add new ones, such as description, tags and overrides.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/14_editor.png" width="900"/>
+
+## **Description**
+
+This is a brief statement describing your item that will be displayed in the marketplace.
+
+## **Overrides**
+
+Overrides determine which Wearable categories or avatar body parts your item will hide. For instance, a hat with attached hair might need to hide the _Hair_ category. A deep-sea diver helmet may require hiding head accessories like earrings, eyewear, tiaras, etc., which wouldn’t be visible. Multiple options can be selected for each override.
+
+- **Base Body**: This refers to core avatar parts like the _head_ and _hands_. For example, if you’re creating a **Handwear** item such as a robot mechanic hand, you’ll likely need to hide _hands_ to prevent overlap and clipping.
+
+- **Wearables**: This includes other Wearable categories. You can hide multiple categories. For more details on each category and how items interact, refer to **[Creating Wearables](https://docs.decentraland.org/creator/wearables/creating-wearables/)**.
+
+{{< hint warning >}}
+Note: The overrides you select will be the suggested default settings for your Wearable. However, users can customize which Wearables are hidden or showing from the Backpack.
+{{< /hint >}}
+
+## **Tags**
+
+Tags are descriptive words that users can utilize for competitions or events!
+
+## **Preview**
+
+To preview your wearable, hover your mouse over the wearable icon on the top left and click on the eye symbol.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/18_preview_smart_wearables_03.gif" />
+
+By clicking on the icon at the lower left you will be able to edit the avatar. This is pretty useful if you have a male and female version of your wearable, so make sure to check how both versions look like in editor, testing different emotes to identify if there are any skinning issues and mixing other wearables to see how it matches with different clothes. When you’re done editing your wearable, click on **_Save_**.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/19_edit_avatar_wearable_03.gif" />
+
+## **Testing in World**
+
+Even after testing the wearable in the editor, it’s important to check how it’s actually going to look like and behave in Decentraland. To test it in world, go to the Collections tab. Select the desired collection and click the button **_See in World_**.
```

Comment:
```suggestion
Even after testing the Wearable in the editor, it’s important to check how it will look and behave in Decentraland. To try it there, go to the Collections tab. Select the desired collection and click the button **_See in Decentraland_**.
```